src/main/java/duke/tasktype/Todo.java

```diff
@@ -0,0 +1,113 @@
+package duke.tasktype;
+
+import duke.Ui;
+import duke.exception.WrongCommandFormatException;
+
+/**
+ * Class to represent a to-do task.
+ * A to-do event does not have any time or date requirements.
+ * It essentially represents a standard task
+ *
+ * @author Houten Teo
+ * @version CS2103T week 3
+ */
+public class Todo implements Task {
```

Comment:
Great to see that you are using inheritance and polymorphism! That should help reduce duplicate code in your codebase and make your project a lot cleaner!